moveit_ros/planning_interface/move_group_interface/src/move_group.cpp

```diff
@@ -590,6 +595,47 @@ class MoveGroup::MoveGroupImpl
     }
   }
 
+  MoveItErrorCode planGraspsAndPick(const std::string &object)
+  {
+    moveit::planning_interface::PlanningSceneInterface psi;
+
+    std::map<std::string, moveit_msgs::CollisionObject> objects = psi.getObjects(std::vector<std::string>(1,object));
+
+    if (objects.size() < 1)
+    {
+      ROS_ERROR_STREAM("Asked for grasps for the object '" << object << "', but the object could not be found");
```

Comment:
hehe, I proposed that too, but actually *none* of the log outputs in this class are `_NAMED` at the moment, and it seemed weird to introduce one occurrence here.

Instead I would propose to name all of them in a later request.